packages/ui/src/components/PostSentenceCreator/PostSentenceCreator.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from "react"
+
+import { HStack, IconButton, Textarea } from "@chakra-ui/react"
+import { useQueryClient } from "@tanstack/react-query"
+import { FaPlus } from "react-icons/fa"
+
+import { useCreateHashtagSentence } from "@wsvvrijheid/services"
+
+type PostSentenceCreatorProps = {
+  hashtagId: number,
+  postId: number,
+  initialContent?: string,
+}
+
+export const PostSentenceCreator = ({ hashtagId, postId, initialContent }: PostSentenceCreatorProps) => {
+  const [value, setValue] = useState(initialContent)
+  const onAddMutation = useCreateHashtagSentence()
+  const queryClient = useQueryClient()
+
+  const handleAdd = () => {
+    onAddMutation.mutate(
+      { hashtagId, value: `${value}::${postId}::${0}::${0}` },
+      {
+        onSuccess: () => queryClient.invalidateQueries({
+          queryKey: ['kv-hashtag-sentences', hashtagId],
+        }),
+      },
+    )
+    setValue('')
+  }
+
+  return (
+    <HStack>
+      <Textarea
+        value={value}
+        onChange={e => setValue(e.target.value)}
+        placeholder="Add sentence"
+        bgColor={'whiteAlpha.700'}
+      />
+      <IconButton
+        aria-label="Add sentence"
+        icon={<FaPlus />}
+        onClick={handleAdd}
+        colorScheme="purple"
```

Comment:
If you don't provide any color scheme, buttons will use primary color as default.

```suggestion
        {...(colorScheme && { colorScheme })}
```